README.md

```diff
@@ -26,31 +26,25 @@ dependencies {
 
 The latest versions can be checked by looking at the [releases](https://github.com/alcatrazEscapee/mcjunitlib/releases) page. As of time of writing (2021-04-17), the latest versions are:
 
-- Minecraft 1.16.5: `1.4.1` (Latest)
+- Minecraft 1.16.5: `1.4.3` (Latest)
```

Comment:
Nitpick, but if you update the latest version, update the date above please.

Reply:
Fixed.

Reply:
Uhhh... it's September tho :P

Reply:
You saw nothing. /s